sdk/servicebus/azservicebus/receiver.go

```diff
@@ -14,11 +24,306 @@ const (
 	SubQueueTransfer   = "transferDeadLetter"
 )
 
-type ServiceBusReceiver struct{}
+type Receiver struct {
+	config struct {
+		ReceiveMode    ReceiveMode
+		FullEntityPath string
+		Entity         entity
+
+		RetryOptions struct {
+			Times int
+			Delay time.Duration
+		}
+	}
+
+	mu       sync.Mutex
+	ns       legacyNamespace
+	receiver internal.LegacyReceiver
+
+	linkState *linkState
+}
+
+type ReceiverOption func(receiver *Receiver) error
```

Comment:
Doc comment

Reply:
Applies to other places with missing doc comments.

Reply:
Added, added, added!